rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -39,9 +39,6 @@ BuildRequires: %{?scl_prefix}ruby(abi) >= %{rubyabi}
 BuildRequires: %{?scl_prefix}rubygems-devel
 BuildRequires: %{?scl_prefix}rubygems
 BuildRequires: foreman-assets >= 1.7.0
-# TODO could be removed later when dependencies are fixed https://github.com/OpenSCAP/foreman_openscap/issues/67
```

Comment:
I don't think this comment was correct actually, they will still be build-deps for asset precompilation.  The tests should confirm!